src/server/api/controllers/members.controller.js

```diff
@@ -57,10 +57,12 @@ const addMember = async (userId, boardId, memberId) => {
   }
   await userIsOwner(userId, boardId);
 
+  const role = userId === memberId ? 'owner' : 'basic';
```

Comment:
With this change, the owner is not added when the board is created

Reply:
ohhhhhhh 🤦🏽 
Yeah. I will make a new issue and PR to change the AddMember logic a bit.

Reply:
Its okay to do it from here, imo.
Since its related to this

Reply:
I think now with changes of AddMember page #251  all should be good 🙏🏽